web/src/stream_create_subscribers_data.js

```diff
@@ -9,10 +9,10 @@ export function initialize_with_current_user() {
     user_id_set.add(current_user_id);
 }
 
-export function sorted_user_ids() {
+export function sorted_users() {
     const users = people.get_users_from_ids(Array.from(user_id_set));
     people.sort_but_pin_current_user_on_top(users);
-    return users.map((user) => user.user_id);
```

Comment:
Why these changes are required? For user settings also, we pass user_ids and not user objects.

Reply:
I didnot look at  implementation of user settings while creating the PR. I have updated the PR and now user_ids are used instead of user objects.